src/bundle/Resources/public/js/scripts/quick.action.manager.js

```diff
@@ -0,0 +1,49 @@
+(function (global) {
+    let actionButtonConfigs = [];
+
+    const registerButton = (config) => {
+        if (!config || !config.container || actionButtonConfigs.some((btn) => btn.id === config.id)) {
+            return;
+        }
+
+        actionButtonConfigs = [...actionButtonConfigs, config].sort((a, b) => a.priority - b.priority);
+        recalculateButtonsLayout();
+    };
+    const unregisterButton = (id) => {
+        actionButtonConfigs = actionButtonConfigs.filter((btn) => btn.id !== id);
+        recalculateButtonsLayout();
+    };
+    const recalculateButtonsLayout = () => {
+        const buttonsToRender = actionButtonConfigs.filter((btn) => {
+            if (typeof btn.checkVisibility === 'function') {
+                const isVisible = btn.checkVisibility();
+
+                return isVisible;
+            }
+
+            return false;
+        });
+
+        buttonsToRender.forEach((buttonConfig, index) => {
+            const { container } = buttonConfig;
+
+            if (!container.style.transition) {
+                container.style.transition = 'all 0.3s ease-in-out';
+            }
+
+            container.style.position = 'fixed';
+            container.style.right = '2rem';
+            container.style.zIndex = buttonConfig.zIndex || 1040;
+
+            const bottomPosition = `${index === 0 ? 2 : index * 3.8 + 2 + index * 0.5}rem`;
```

Comment:
```suggestion
            const bottomPosition = `${index === 0 ? 2 : index * 4.3 + 2}rem`;
```
where does this number comes from by the way?